CMakeLists.txt

```
@@ -14,7 +14,7 @@ option(ARMA_EXTRA_DEBUG "Compile with extra Armadillo debugging symbols." OFF)
 option(TEST_VERBOSE "Run test cases with verbose output." OFF)
 option(BUILD_TESTS "Build tests." ON)
 option(BUILD_CLI_EXECUTABLES "Build command-line executables." ON)
-option(DISABLE_DOWNLOADS "Disable downloads of dependencies during build." OFF)
+option(DOWNLOAD_DEPENDENCIES "Download of dependencies during build." OFF)
```

```suggestion
option(DOWNLOAD_DEPENDENCIES "Automatically download dependencies if not available." OFF)
```
Just a minor clarification.